docker-build/Dockerfile

```diff
@@ -1,57 +1,28 @@
-FROM        debian 
-
-ARG         gsd
-ENV         GITHUB_STABLE_DIFFUSION $gsd
-
-ARG         rsd
-ENV         REQS $rsd
-
-ARG         cs
-ENV         CONDA_SUBDIR $cs
-
-ENV         PIP_EXISTS_ACTION="w"
-
-# TODO: Optimize image size
-
-SHELL       ["/bin/bash", "-c"]
-
-WORKDIR     /
-RUN         apt update && apt upgrade -y \
-            && apt install -y \
-            git \
-            libgl1-mesa-glx \
-            libglib2.0-0 \
-            pip \
-            python3 \
-            && git clone $GITHUB_STABLE_DIFFUSION
-
-# Install Anaconda or Miniconda
-COPY        anaconda.sh .
-RUN         bash anaconda.sh -b -u -p /anaconda && /anaconda/bin/conda init bash
-
-# SD 
-WORKDIR     /stable-diffusion 
-RUN         source ~/.bashrc \
-            && conda create -y --name ldm && conda activate ldm \
-            && conda config --env --set subdir $CONDA_SUBDIR \
-            && pip3 install -r $REQS \   
-            && pip3 install basicsr facexlib realesrgan \          
-            && mkdir models/ldm/stable-diffusion-v1 \
-            && ln -s "/data/sd-v1-4.ckpt" models/ldm/stable-diffusion-v1/model.ckpt 
-
-# Face restoreation
-# by default expected in a sibling directory to stable-diffusion
-WORKDIR     /       
-RUN         git clone https://github.com/TencentARC/GFPGAN.git 
-
-WORKDIR     /GFPGAN 
-RUN         pip3 install -r requirements.txt \
-            && python3 setup.py develop \
-            && ln -s "/data/GFPGANv1.4.pth" experiments/pretrained_models/GFPGANv1.4.pth
-
-WORKDIR     /stable-diffusion 
-RUN         python3 scripts/preload_models.py         
-
-WORKDIR     /
-COPY        entrypoint.sh .
-ENTRYPOINT ["/entrypoint.sh"]
\ No newline at end of file
+FROM nvidia/cuda:11.7.1-base-ubuntu22.04
+FROM continuumio/miniconda3
+
+RUN apt update
+RUN apt install dos2unix -y
+
+WORKDIR /
+COPY . /InvokeAI/
+
+WORKDIR /InvokeAI
+
+RUN mkdir -p models/ldm/stable-diffusion-v1
+#RUN pwd
+#RUN ls -la
+RUN mv model.ckpt models/ldm/stable-diffusion-v1/model.ckpt
```

Comment:
Why do you "RUN" and then mv the model.ckpt (and where does it come from?), wouldn't this be provided from your Client which then could be "COPY"-ed instead?

Reply:
The file is actually copied earlier, this was more just to get it in the right place where invoke.py expects it. In the future I'm going to change this to something that is just volume mapped in so it can be swapped without rebuilding every time.

Reply:
Yeah, that's what I meant, sorry if I said it in a confusing way.

Reply:
Oh and to answer the question the assumption is that the model would just be placed into the root before building. It was a quick solution to make sure everything was working.

Reply:
So why do you first copy something into the container and then move it inside of the container again? Are you aware how SquashFS is working and what to avoid when building a container-image?